From: Ruben Undheim <ruben.undheim@gmail.com>
Date: Tue, 14 Jun 2016 18:46:08 +0200
Subject: This makes sure that the CXXFLAGS given by the Debian build system
 are not overwritten,
 but prepended. Additionally unwanted CXXFLAGS are removed.

---
 icebram/Makefile  | 3 ++-
 icemulti/Makefile | 2 +-
 icepack/Makefile  | 2 +-
 icepll/Makefile   | 2 +-
 iceprog/Makefile  | 4 ++--
 icetime/Makefile  | 2 +-
 6 files changed, 8 insertions(+), 7 deletions(-)

--- a/config.mk
+++ b/config.mk
@@ -1,8 +1,8 @@
 CXX ?= clang++
 CC ?= clang
 LDLIBS = -lm -lstdc++
-CFLAGS = -MD -O0 -ggdb -Wall -std=c99 -I/usr/local/include
-CXXFLAGS = -MD -O0 -ggdb -Wall -std=c++11 -I/usr/local/include
+CXXFLAGS += -Wall -std=c++11
+CFLAGS += -Wall -std=c99
 PKG_CONFIG ?= pkg-config
 DESTDIR ?=
 PREFIX ?= /usr/local
